02-prompting-101/verifying-information-from-chatGPT.md

```diff
@@ -0,0 +1,150 @@
+
+The issue of generating non-existent citations is widespread enough that some universities have statements [like this one from University of Arizona](https://ask.library.arizona.edu/faq/387173) on what to do when someone can't find the citations ChatGPT provided.
+
+### Inclusion of information from biased sources
+
+Not all sources should be given the same credence. A resource or article that a tool mentions actually existing does not mean that it’s a trustworthy source of information. There can be a vast amount of biased information in the datasets which LLMs are trained on, meaning we need to take a critical lens to the responses AIs generate to ensure we aren’t propagating harmful biases using what they’ve generated.
+
+## How Can We Validate?
+
+So, if the AI tool doesn’t know the latest news, can’t tell us where its training data came from, or it isn’t capable of looking the facts up online itself, how can we go about checking information in the response?
+
+Fact checking! We’ll use our searching powers that we’ve honed looking up documentation and tutorials to take a critical eye to any information in the response that can be verified such as:
+- proper names
+- place names
+- references to time, distance, date, season
+- physical descriptions
+- references to the gender of anyone described 
+- quotations (and facts within quotes)
+- any argument or narrative that depends on fact
+
+We want to cross-check details of the response with other reliable sources of information. We may need to do several searches, and our fact checking could include consulting multiple reputable websites, academic publications, or experts in the topic. Depending on the subject matter we can pull in other resources like databases of news and journal articles (such as LexisNexis or ScienceDirect), and tools like Google Scholar.
+
+When a generative AI tool makes a claim like "Ida B. Wells became editor and co-owner of the newspaper 'The Free Speech and Headlight' in 1889" we can look for multiple sources that confirm whether: 
+- Ida B. Wells existed
+- The newspaper 'The Free Speech and Headlight' existed
+- If she became the editor and co-owner in 1889 
+
+### Looking for SMART resources
+
+When doing our own searching, we also need to be asking ourselves if the sources we’re using to verify information are trustworthy. Anyone can publish anything on the internet, but there are frameworks we can use to ask ourselves questions about a website or source to help us investigate if it may be worthwhile. Once such framework is the SMART Check:
```

Comment:
```suggestion
When doing our own searching, we also need to be asking ourselves if the sources we’re using to verify information are trustworthy. Anyone can publish anything on the internet, but there are frameworks we can use to ask ourselves questions about a website or source to help us investigate if it may be worthwhile. One such framework is the SMART Check:
```